cmd/program/program.go

```diff
@@ -128,6 +166,31 @@ func (p *Project) CreateMainFile() error {
 		}
 	}
 
+	if p.DBDriver != "None" {
```

Comment:
We should probably also integrate the NEW CLI flag feature here as well from this PR:
https://github.com/Melkeydev/go-blueprint/pull/18

i will let you decide if you want to integrate this into this PR or a separate one!